e2e/DeviceLocked.test.js

```diff
@@ -0,0 +1,35 @@
+const Utils = require('./Utils');
+const Android = require('./AndroidUtils');
+const testIDs = require('../playground/src/testIDs');
+const exec = require('shell-utils').exec;
+const _ = require('lodash');
+
+const { elementByLabel, elementById, sleep } = Utils;
+const IS_RELEASE = _.includes(process.argv, '--release');
+const KEY_CODE_R = 46;
+
+describe('application lifecycle test', () => {
+  beforeEach(async () => {
+    await device.relaunchApp();
+  });
+
+  test('launch from locked screen', async () => {
+    await device.terminateApp();
+    await Android.pressKeyCode('26')
+    await Android.pressKeyCode('26')
+    await device.launchApp();
+    await Android.executeShellCommand(`input touchscreen swipe 300 1200 500 0 100`)
```

Comment:
What does this line do? I assume it's used to swipe across the screen to open the device, if so lets extract it to a function with a meaningful name

Reply:
yes, the `input touchscreen` command it to perform a swipe on the device such that the device asks the PIN to unlock it.